app/assets/javascripts/components/nav_menu.js.jsx

```diff
@@ -0,0 +1,34 @@
+var React = require('react');
+
+var NavMenu = React.createClass({
+
+  getInitialState: function(){
+    return { focused: 0 };
```

Comment:
Use a more descriptive name, like `currentlyFocusedIndex`. Also, try to use whitespace consistently.